ui/src/app/pages/access/access-renewal.spec.tsx

```diff
@@ -29,7 +29,10 @@ describe('Access Renewal Page', () => {
     const {profile} = profileStore.get();
 
     const newProfile = fp.set('accessModules',
-        {modules: accessRenewalModules.map(m => ({moduleName: m, expirationEpochMillis: expiredTime}))},
```

Comment:
expiration without completion is not a valid state, so we now add completion times here too